Sentence casing (#1037)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab.md

---
title: "Model timing tab"
id: "cloud-model-timing-tab"

> description: "Visually explore runs with the model timing dashboard."

Reviewer: ```suggestion
description: "Visually explore runs and identify long-running models with the model timing dashboard."
```

> description: "You can use the model timing dashboard to visually explore runs."

Author: ```suggestion
description: "Visually explore runs with the model timing dashboard."
```
